src/main/scala/mesosphere/marathon/core/launcher/impl/InstanceOpFactoryImpl.scala

```diff
@@ -146,6 +146,7 @@ class InstanceOpFactoryImpl(
      */
 
     def maybeLaunchOnReservation: Option[OfferMatchResult] = if (needToLaunch) {
+      logger.info("Need to launch on reservation.")
```

Comment:
wouldn't it help to also say for which runspec? When there are many apps on the cluster it might not be concrete enough... ?

Reply:
![Well, yeah](https://media.giphy.com/media/26BRrSvJUa0crqw4E/giphy.gif)